setup.cfg

```diff
@@ -0,0 +1,13 @@
+[metadata]
+name = torchboard
+version = 0.1.0
+description = "Clumsy TensorBoard copycat for PyTorch"
+author = "Igor Andruskiewitsch", "Marco Martinelli"
+author_email = "i.andruskiewitsch23@gmail.com", "marcocba@gmail.com"
+license = "MIT"
+readme = "README.md"
+python = "^3.8"
+repository = "https://github.com/TorchBoard/TorchBoard"
+
+[options]
+packages = find:
```

Comment:
This is pip-installing both `torchboard` and `tests` as package, we should specify that the source folder is `torchboard` only!